src/components/templates/FireBarrel/FireBarrel.tsx

```diff
@@ -34,17 +34,17 @@ const FireBarrel: React.FC = () => {
 
   const filterPartygoers = (
     venue: Venue,
-    partygoers: readonly WithId<User>[]
+    recentWorldUsers: readonly WithId<User>[]
   ): WithId<User>[] =>
-    partygoers?.filter((person) => person.room === venue?.name);
+    recentWorldUsers?.filter((person) => person.room === venue?.name);
 
   useEffect(() => {
     if (venue) {
-      const partyPeople = filterPartygoers(venue, partygoers);
+      const partyPeople = filterPartygoers(venue, recentWorldUsers);
 
       setCurrentPartygoers(partyPeople);
     }
-  }, [partygoers, venue]);
+  }, [recentWorldUsers, venue]);
```

Comment:
This would probably be resolved by a small hook such as `recentUsersInRoom(venueId, roomName)`, which would probably consume `recentUsersInVenue(venueId)`